restapi/src/main/java/com/craftsmanship/tfm/restapi/config/PersistenceConfig.java

```diff
@@ -24,7 +25,12 @@
   private int serverPort;
 
   @Bean
-  public ItemPersistenceGrpc persistenceHandler() {
+  public ItemPersistenceGrpc itemPersistenceHandler() {
     return new ItemPersistenceGrpc(serverHost, serverPort);
   }
+
+  @Bean
+  public OrderPersistenceGrpc orderPersistenceHandler() {
```

Comment:
Would not be better OrderPersistence ?

Reply:
I wanted to let clear that in production OrderPersistenceGrpc will be used. But I can change if you want.